TESTING.md

```diff
@@ -51,15 +50,19 @@ Prerequisite: You have to **sign out** from all existing accounts.
 * IDE should receive a valid token automatically.
 * `Commands` and `Chat` tabs are ready to use.
 
-### Sign in with GitLab
+Verify the remaining SSO methods by performing the same steps for `Sign in with GitLab` and `Sign in with Google`.
 
-Onboarding through GitLab is similar to [Sign in with GitHub](#sign-in-with-github), and the authorization is also done
-through the browser. Expected behaviour is identical.
+### Remove all accounts
 
-### Sign in with Google
+Prerequisite: You have to be **signed in**. This is important because we expect certain components to be refreshed automatically.
 
-Onboarding through GitLab is similar to [Sign in with GitHub](#sign-in-with-github), and the authorization is also done
-through the browser. Expected behaviour is identical.
+1. Navigate to `Settings` > `Sourcegraph & Cody`.
+2. Remove all accounts and apply settings.
+
+#### Expected behaviour
+
+* Cody toolbar is automatically refreshed and the user is greeted with the `Welcome to Cody` panel.
```

Comment:
nit: `s/toolbar/tool window/g`

Reply:
Sorry, but what do you mean by `s/toolbar/tool window/g`? :smile:

Reply:
It's sed replace syntax, it means you have typo or other text bug which reviewer is asking you to fix ;)

Reply:
I took a liberty to merge it, we can improve it next time.